src/containerbuddy/config.go

```diff
@@ -27,6 +27,7 @@ type ServiceConfig struct {
 	Port             int    `json:"port"`
 	TTL              int    `json:"ttl"`
 	IsPublic         bool   `json:"publicIp"` // will default to false
```

Comment:
I commented in https://github.com/joyent/containerbuddy/issues/20#issuecomment-161279931 to this effect, but if we're going to have an `Interface` option then we've obviated the need to have the `IsPublic` option. Let's get rid of that in this same PR.